src/app/components/PortraitVideoCarousel/PortraitVideoCarouselNavigation/index.tsx

```diff
@@ -61,23 +63,23 @@ export default ({ scrollPaneRef }: PortraitVideoCarouselNavigationProps) => {
       <div css={styles.buttonGroup}>
         <button
           type="button"
-          aria-label={scrollLeftAria}
+          aria-label={previousAria}
           onClick={() => scroll(dir === 'ltr' ? 'left' : 'right')}
           disabled={!canScrollLeft}
```

Comment:
Can we use an is convention here just to match the rest of the codebase, something like isScrollableLeft/isScrollableRight?